src/General/Badge/BadgeStyle.ts

```diff
@@ -1,45 +1,49 @@
 import styled from 'styled-components';
-import { PrimaryColor, Greyscale } from '../../Utils/Colors';
+import { Greyscale, SecondaryColor } from '../../Utils/Colors';
 import { BadgeVariant } from './BadgeVariant';
 
 export const BadgeContainer = styled.div<BadgeContainerProps>`
   position: relative;
   display: inline-flex;
-  background: ${({ variant }) =>
-    variant === BadgeVariant.DEFAULT
-      ? `${PrimaryColor.glintsred}`
-      : `${Greyscale.grey}`};
-  color: ${Greyscale.white};
-  border-radius: 10px;
-  margin: 0 5px;
-
-  ${({ sup }) => {
-    if (sup) {
-      return `
-        top: -.6em;
-      `;
+  ${({ variant }) => {
+    switch (variant) {
+      case BadgeVariant.BLUE:
+        return `
+          background: ${SecondaryColor.actionblue};
+        `;
+      case BadgeVariant.DIMMED:
+        return `
+          background: ${Greyscale.devilsgrey};
+        `;
+      case BadgeVariant.WHITE:
+        return `
+          background: ${Greyscale.white};
+        `;
+      default:
+        return `
+          background: #ed9300;
+        `;
     }
   }}
+  color: ${({ variant }) =>
+    variant === BadgeVariant.WHITE
+      ? `${SecondaryColor.actionblue}`
+      : `${Greyscale.white}`};
+  border-radius: 3px;
+  margin: 0 4px;
+  padding: 0 4px;
```

Comment:
Should we use `${Spacing.4}` here? `import { Spacing } from 'src/Layout/Spacing'`